.cargo/config.toml

```diff
@@ -2,8 +2,8 @@
 CARGO_WORKSPACE_DIR = { value = "", relative = true }
 
 [alias]
-pico = ["run", "-p", "xtask", "--", "pico", "--"]
-risc0 = ["run", "-p", "xtask", "--", "risc0", "--"]
-sp1 = ["run", "-p", "xtask", "--", "sp1", "--"]
-zisk = ["run", "-p", "xtask", "--", "zisk", "--"]
-zkm = ["run", "-p", "xtask", "--", "zkm", "--"]
+pico = ["run", "-p", "xtask", "--", "pico"]
+risc0 = ["run", "-p", "xtask", "--", "risc0"]
+sp1 = ["run", "-p", "xtask", "--", "sp1"]
+zisk = ["run", "-p", "xtask", "--", "zisk"]
+zkm = ["run", "-p", "xtask", "--", "zkm"]
```

Comment:
I removed this since now `xtask` accepts an optional flag `--manifest-folder`.
For SP1, this is set explicitly since we want to target a specific guest program crate instead of the workspace. If not provided, it defaults to the `ere-guest` workspace (same as before).

The `--` now is mainly a responsibility of whoever wants to pass arguments to cargo (if any, I don't think we rely on this today).

I think at some point might be good to do the patching via libs instead of `cargo -p xtasks`? As in, `ere-hosts` using `xtasks` as a library instead of via `cargo`.

In any case, no need to really do that if mid-term the way patching works works and we don't need to do them in crates (it might take a while though). 

No strong opinions -- I think we can leave it this way.